summarizer_server/server.py

```diff
@@ -66,13 +66,14 @@ def submit_feedback():
     gaia = request_payload.get("gaia", "")
 
     # check if user wishes to be anonymous
-    user = userservice.get_anonymous()
     if email != "" and gaia != "":
         user = userservice.get(email)
```

Comment:
I think you can just change this to `user = userservice.get_or_create(email)` now.

Reply:
Right right, this is a lot better, thanks! Done!